Root/VariableDefinitions.cxx

```diff
@@ -261,14 +261,14 @@ int VD::ttbarHF_ext(const xAOD::EventInfo* ei){
 }
 
 bool VD::topTag(const xAOD::Jet* jet, std::string wp, float maxAbsEta, float minPt){
-  if(std::fabs(jet->eta()) > maxAbsEta) return false;
+  if(std::fabs(jet->eta()) >= maxAbsEta) return false;
```

Comment:
Oof, that's strange logic. I always include inclusively and exclude exclusively.